rawfile/utils/logs.py

```diff
@@ -113,27 +113,29 @@ def wrap(self, request, context):
         is_json = format == LoggingFormats.JSON
         args = {
             "handler": func.__name__,
-            "request": MessageToDict(request) if is_json else request,
             "starttime": start,
         }
+        res = None
         try:
             res = func(self, request, context)
             end = datetime.now()
             args.update(
                 {
-                    "response": MessageToDict(res) if is_json else res,
                     "latency": end - start,
                     "endtime": end,
                     "success": True,
                 }
             )
-            logger.info("GRPC Server Access Log", **args)
+            logger.success("GRPC Server Access Log", **args)
             return res
         except Exception as exc:
             end = datetime.now()
             args.update(
                 {
-                    "response": None,
+                    "response": (MessageToDict(res) if is_json else res)
```

Comment:
can we even has res here?

Reply:
If other parts have failed after calling the handler, it should be less than 1% I think but I added to be ensured that we cover that case